aptos-move/framework/aptos-stdlib/sources/from_bcs.spec.move

```diff
@@ -14,6 +14,13 @@ spec aptos_std::from_bcs {
         axiom<T> forall b1: vector<u8>, b2: vector<u8>:
             (deserialize<T>(b1) == deserialize<T>(b2) ==> b1 == b2);
 
+        // If the input are equal, the result of deserialize should be equal too
```

Comment:
This was disabled due to a timeout. Does it work now?

Reply:
These two axioms are new. Also, Xudong's previous PR already added this.